build-constraints.yaml

```diff
@@ -1,6 +1,6 @@
 ghc-major-version: "9.0"
```

Comment:
Shoud that be `"9.2"`?

Reply:
yes, but I think this field is no longer in use by our tooling.